GitExtensionsVSIX/source.extension.vsixmanifest

```diff
@@ -13,7 +13,9 @@
     </Metadata>
 
     <Installation>
-        <InstallationTarget Version="[14.0,15.0)" Id="Microsoft.VisualStudio.Community" />
+        <InstallationTarget Version="[14.0,16.0)" Id="Microsoft.VisualStudio.Community" />
```

Comment:
I faced with similar problem and quickly fix it by using NuGet package instead of COM interop in 8b63e2f2d1e28c5b67fc88eb9cd8849b1b225250